### PR TITLE
Not great

### PR DESCRIPTION
**Pull request by FayeCarter**

### CODE REVIEW

File: TodoList.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+```
+As a busy politician
+I can see all of today's headlines in one place
+So I know what the big stories of the day are
+```
+- [X] Add api getter
+- [X] Add Controller
+- [X] Stub out API
+- [X] Add Article Object
+- [X] Create Content
```
Comment (grahamfalconer) [inline TodoList.md:15]: Nice clear goals, lgtm

File: depreciated/scripts.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+request.send()
+
+const app = document.getElementById('root')
+
+const container = document.createElement('div')
+container.setAttribute('class', 'container')
+
+app.appendChild(container)
+
```
Comment (grahamfalconer) [inline depreciated/scripts.js:49]: Nice use of Vanilla JS 😺

File: lib/models/api.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  var result = new XMLHttpRequest();
+  result.open( "GET", url, false ); // false for synchronous request
+  result.send();
+  data = JSON.parse(result.responseText);
+  data.response.results.forEach( i => {
+    this.results.push(i.fields)
+  })
+  return this.results
+}
+
```
Comment (grahamfalconer) [inline lib/models/api.js:15]: LGTM

File: depreciated/style.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+a {
+  margin: 0;
+  text-decoration: none;
+  color: #666;
+}
+.card:hover {
+  box-shadow: 2px 8px 45px rgba(0, 0, 0, .15);
+  transform: scale(1.05);
+}
```
Comment (grahamfalconer) [inline depreciated/style.css:68]: ^ looks excellent 👍

File: lib/controller/newspaper.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+  return this.articles
+}
+
+Newspaper.prototype.createArticles = function() {
+  this.content.forEach(story => {
+    article = new Article()
+    article.fill(story)
+    this.articles.push(article)
+  })
```
Comment (grahamfalconer) [inline lib/controller/newspaper.js:19]: Nice funtion 🧙‍♀️

File: lib/models/api.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+guardianAPI.prototype.makeRequest = function(url) {
+  var result = new XMLHttpRequest();
+  result.open( "GET", url, false ); // false for synchronous request
+  result.send();
+  data = JSON.parse(result.responseText);
+  data.response.results.forEach( i => {
+    this.results.push(i.fields)
+  })
```
Comment (grahamfalconer) [inline lib/models/api.js:12]: Glad you cracked this chestnut 🍘

File: lib/models/article.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+function Article() {
+};
+
+Article.prototype.fill = function(content) {
+  this.headline = content.headline;
+  this.body = content.bodyText;
+  this.image = content.thumbnail;
+  this.url = content.shortUrl;
+  this.full = content.body;
+}
```
Comment (grahamfalconer) [inline lib/models/article.js:10]: Could 'full' be named something better

File: lib/views/interface.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    container.href = this.url
+    
+    container.appendChild(link)
+    link.appendChild(card)
+    card.appendChild(h2)
+    card.appendChild(img)
+    card.appendChild(p)
+    app.appendChild(container)
+  })
+})()
```
Comment (grahamfalconer) [inline lib/views/interface.js:35]: LGTM

File: lib/models/article.js (hunk 7 above)
Comment (FayeCarter) [reply lib/models/article.js:10]: Good point - thank you

**Pull request closed without merging**